style.css

```diff
@@ -389,3 +425,245 @@ textarea {
+
+  header nav .toolbar ul li {
+    display: flex;
+    align-items: center;
+    padding: 8px 34px;
+  }
+
+  header nav .toolbar ul li a {
+    color: #fff;
+    text-decoration: none;
+  }
+
+  .menu-bar img {
+    display: none;
+  }
+
+  .headline {
+    display: flex;
+    padding-bottom: 150px;
+    background-position: 100% 80%;
+    background-size: 558px;
+  }
+
+  .headline-body {
+    width: 556px;
+    padding-right: 400px;
+    order: 99;
+  }
+
+  .headline-body h1 {
+    width: 60%;
+    margin-bottom: 0;
+    margin-top: 0;
+  }
+
+  .headline-body p,
+  .headline h3 {
+    width: 75%;
+  }
+
+  .headline-body p {
+    padding-top: 0;
+  }
+
+  .headlink {
+    margin: 0 24px;
+  }
+
+  .headlink .social-links {
+    display: block;
+  }
+
+  .work-container {
+    padding-bottom: 128px;
+    grid-template-columns: auto auto auto;
+    gap: 3% 3%;
+  }
+
+  .item {
+    background-position: center;
+    background-size: cover;
+    background-repeat: no-repeat;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .work-item-1 {
+    grid-column: 1/4;
+    padding-top: 12px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  .no-btn button {
+    display: none;
+  }
+
+  .item:nth-child(2) {
+    background-image: url(image/Img_Placeholder_2.svg);
+    display: flex;
+    position: relative;
+    height: 450px;
+  }
+
+  .work-item-2 .project-btn {
+    position: absolute;
+    bottom: 0;
+  }
+
+  .item:nth-child(3) {
+    background-image: url(image/Img_Placeholder_3.svg);
+  }
+
+  .item:nth-child(4) {
+    background-image: url(image/Img_Placeholder_4.svg);
+  }
+
+  .item:nth-child(5) {
+    background-image: url(image/Img_Placeholder_5.svg);
+  }
+
+  .item:nth-child(6) {
+    background-image: url(image/Img_Placeholder_6.svg);
+  }
+
+  .item:nth-child(7) {
+    background-image: url(image/Img_Placeholder_7.svg);
+  }
+
+  .work-item-1 img,
+  .work-item-1 .write-up {
+    flex-basis: 49%;
+  }
+
+  .item-display {
+    display: none;
+  }
+
+  .abt-sct {
+    margin-top: 0;
+  }
+
+  .work-section-container .heading {
+    width: 90%;
+    margin-left: 12px;
+    display: flex;
+    align-items: center;
+  }
+
+  .work-section-container .heading h2 {
+    margin-right: 0;
+    width: 365px;
+  }
+
+  .contact hr {
+    width: 100%;
+  }
+
+  .work-section-container .heading hr {
+    align-self: flex-end;
+    margin-left: 0%;
+    margin-top: 0;
+    height: 3px;
+    border: 1px solid none;
+    background-color: #212121;
+  }
+
+  .about-section-container {
+    background-image: url("image/background_img/desk_Group.svg");
+    background-size: 508px;
+    padding-top: 100px;
+    background-position: 118% 12%;
+  }
+
+  .about-me {
+    text-align: left;
+    max-width: 556px;
+    margin-bottom: 120px;
+  }
+
+  .left {
+    margin-left: 204px;
+  }
+
+  .about-me p {
+    padding-right: 124px;
+    font-size: 20px;
+  }
+
+  .about-me h1 {
+    text-align: left;
+  }
```

Comment:
- [ ] Please, kindly fix your about me section so that the paragraph takes up almost the middle of the section to be close to this Figma design.

**FIGMA**

![about](https://user-images.githubusercontent.com/59397159/165848515-a3a4f4c6-58f0-4580-bbae-71b032cf1385.png)

**YOUR DESIGN**

![YOU](https://user-images.githubusercontent.com/59397159/165848716-4cd25290-eb4f-42dc-90bf-16a8b98ecfc7.png)